packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -114,6 +114,32 @@ export default class Reachability extends EventsScope {
     this.clusterReachability = {};
   }
 
+  /**
+   * Fetches the list of media clusters from the backend
+   * @param {boolean} isRetry
+   * @private
+   * @returns {Promise<{clusters: ClusterList, joinCookie: any}>}
+   */
+  async getClusters(isRetry = false): Promise<{clusters: ClusterList; joinCookie: any}> {
+    try {
+      const {clusters, joinCookie} = await this.reachabilityRequest.getClusters(
+        MeetingUtil.getIpVersion(this.webex)
+      );
+
+      return {clusters, joinCookie};
+    } catch (error) {
+      if (isRetry) {
+        throw new Error('Failed to get clusters list');
```

Comment:
might be better to throw the original `error` as it will potentially have more useful info like the error from the backend or a tracking id or something

Reply:
Done